src/shared/config.ts

```diff
@@ -158,7 +159,7 @@ const ConvictConfig = Convict<ServiceConfig>({
 // Load and validate general config based on environment variable
 const env = ConvictConfig.get('ENV')
 
-ConvictConfig.loadFile(`${__dirname}/../../config/${env}.json`)
+ConvictConfig.loadFile(path.join(__dirname, `/../../config/${env}.json`))
```

Comment:
@lewisdaly Maybe we should drop using multi-stage config support as we did for thirdparty-api-adapter ? If yes then let do it in a separate story.